lib/led.js

```diff
@@ -23,27 +143,36 @@ function Led(opts) {
   var pinValue;
   var defaultLed;
   var state;
-  var isFirmata = true;
 
   if (!(this instanceof Led)) {
     return new Led(opts);
   }
 
+  var controller = null;
+  var err = null;
+
+  if (!opts || !opts.hasOwnProperty("controller")) {
+    controller = Controllers["Standard"];
+  }
+  else if (typeof opts.controller === "string") {
+    controller = Controllers[opts.controller];
+  }
+  else {
+    controller = opts.controller;
+  }
+
+  if (controller == null) {
+    controller = Controllers["Standard"];
+  }
+
+  Object.defineProperties(this, controller);
```

Comment:
@dtex `Led` can be initialized with no parameters, since there is a `defaultLed` hardcoded to pin 13. I can definitely see the reasoning behind this, since Arduino has the LED on pin 13, and it makes the "hello world" dead simple.